kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilder.scala

```diff
@@ -44,6 +45,8 @@ class SparkProcessBuilder(
 
   import SparkProcessBuilder._
 
+  var yarnClient = YarnClient.createYarnClient
```

Comment:
```suggestion
  val yarnClient = getYarnClient
  
  def getYarnClient: YarnClient = YarnClient.createYarnClient
```
In the test, you could override the `getYarnClient` method.